package.json

```diff
@@ -17,6 +17,7 @@
     "dev": "concurrently --kill-others-on-fail \"yarn client\" \"yarn server\""
   },
   "dependencies": {
+    "caniuse-lite": "^1.0.30001240",
```

Comment:
Was this addition intentional?

Reply:
Not too sure, as on initial run yarn dev threw some errors and told me to update caniuse-lite :/

Reply:
Ah yep, all good